datastore/src/main/java/io/spine/server/storage/datastore/DsAggregateStorage.java

```diff
@@ -198,15 +223,16 @@ public int compare(AggregateEventRecord o1, AggregateEventRecord o2) {
     }
 
     /**
-     * Generates an identifier of the Datastore record basing on the given {@code Aggregate} identifier.
+     * Generates an identifier of the Datastore record basing on the given {@code Aggregate}
+     * identifier.
      *
      * @param id an identifier of the {@code Aggregate}
      * @return the Datastore record ID
      */
     protected DatastoreRecordId generateDatastoreId(I id) {
```

Comment:
It looks like a conversion method. We do not really generate inside. We convert. A better name would be `toDatastoreId()` or `toRecordId()`.

Assuming we are under `datastore` package, the returned type can be also named `RecordId`.